src/app/main.cpp

```diff
@@ -495,6 +498,8 @@ int main( int argc, char *argv[] )
   // This behavior is used to load the app, snapshot the map,
   // save the image to disk and then exit
   QString mySnapshotFileName = QLatin1String( "" );
+  QString configLocalStorageLocation = QLatin1String( "" );
```

Comment:
Not directly a result of your work - but these `QLatin1String("")`s do nothing and can be removed.